src/controllers/timeTaskCtrl.js

```diff
@@ -0,0 +1,112 @@
+const TimeTask = require('../models/timeTaskModel');
+
+exports.createTimeTask = (req, res) => {
```

Comment:
I would have seen more a method to start a task that takes care of all the logic behind it and another method to finish the task so the front end doesn't have to go through anything?
@kilianpichard  / @BrystoQ  ?